docs/suh-template/plan/20260511_844_가격_입력_적정가격_화면_이동.md

```
@@ -0,0 +1,219 @@
+- 위치 선택(`ItemRegisterLocationScreen`)과 동일한 패턴 — 등록 폼에서 세부 입력 화면으로 이동 후 결과 반환
+- 숫자 키보드 자동 포커스 시 전체 화면이 안전 (BottomSheet는 키보드 + 시트 높이 충돌 가능)
+- 디자인 목업이 전체 화면 구조로 확정됨
+
+**대안 — BottomSheet**:
+- 선택하지 않은 이유: 키보드 올라올 때 레이아웃 overflow 처리가 복잡하고, 디자인 목업이 전체 화면 구조임
+
+---
+
+## 5. 화면 구조 (디자인 목업 기준)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)